components/Buttons/tests/unit/ButtonsThemingTest.swift

```diff
@@ -162,34 +162,34 @@ class ButtonsThemingTest: XCTestCase {
     let customColor = UIColor.orange
     colorScheme.primaryColor = customColor
     scheme.colorScheme = colorScheme
-
+    
     // When
     button.applyTextTheme(withScheme: scheme)
-
+    
     // Then
     XCTAssertEqual(button.titleColor(for: .normal), customColor)
     XCTAssertEqual(button.imageTintColor(for: .normal), customColor)
   }
-
+  
```

Comment:
Oh no! White space!

Reply:
Swift file!
whitespaces removed in https://github.com/material-components/material-components-ios/pull/5869/commits/188c094c4a58a41e9364b6f2699b8c48fde8ad77